test/custom-validation.spec.js

```diff
@@ -38,7 +38,6 @@ describe('Custom validation constraint:', () => {
 
     const expectedCurrentItemEntry = {
       itemValue: doc.baseProp.customValidationProp,
-      oldItemValue: null,
```

Comment:
Since properties that are `undefined` are no longer serialized as the string "null" by `json-stringify-module`, this property no longer needs to exist to simulate that case when the object is serialized by `JSON.stringify` below.